### DISCUSSION TITLE
withdraws

### DISCUSSION THREAD

**Discussion by DmitriKivilev** [Chia-Network/chia-blockchain#14399] [Support]
Hello, I made a withdrawal of Xch funds to the address xch1qjy4nf4f0m3x59qjvra46zj0gfxc6dqswftgn0cly22zltvpdh4slkhhj3 . No coins arrived at the address, there is no entry in the blockchain. In 2022, I made a withdrawal of funds at this address and the coins came immediately and there is a record in the blockchain.

where did i go wrong?

**Comment by shaneo257** [comment]
https://www.spacescan.io/address/xch1qjy4nf4f0m3x59qjvra46zj0gfxc6dqswftgn0cly22zltvpdh4slkhhj3 shows a received tx from yesterday, as well as the one from 2022
